strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/npm/NpmPackageController.java

```diff
@@ -95,7 +94,7 @@ public void download(@PathVariable(name = "storageId") String storageId,
     {
         Repository repository = getRepository(storageId, repositoryId);
         RepositoryPath path = npmLayoutProvider.resolve(repository, URI.create(resource));
-        ArtifactInputStream is = (ArtifactInputStream) npmArtifactManagementService.resolve(storageId,
+        InputStream is = npmArtifactManagementService.resolve(storageId,
```

Comment:
Cosmetic: Could you please reformat the parameters as well? Thanks! :)

Reply:
ok

Reply:
fixed